### ISSUE TITLE
Coastlines

### ISSUE THREAD

**Issue by beelsebob** [beelsebob/OpenStreetPad#5]
Description:
OpenStreetPad currently has no reasonable way to deal with natural=coastline ways that delimit landmass.

Steps to reproduce:
1) scroll to an area with coast line in it.

Expected results:
The sea and land are correctly rendered.

Actual results:
There is no way to tell sea and land apart.

**Comment by claudiush** [comment]
Quoting from http://wiki.openstreetmap.org/wiki/Coastline :
"The sea is delineated using natural=coastline with the direction of coastline organised so that land is always on the left of the way in the direction it is drawn."

Should you be able to draw land always to the right of a coastline way and closing it as a polygon at the viewport border?

**Comment by beelsebob** [comment]
Yep, that works, as long as there's at least one coastline way in the area being rendered.  The bigger problem is how to deal with polygons with no coastline – are they under water, or on land?
